packages/front-end/src/app/pages/labs/index.vue

```diff
@@ -1,12 +1,11 @@
 <script setup lang="ts">
-  const $route = useRoute();
   const $router = useRouter();
 </script>
 
 <template>
   <EGPageHeader title="Labs" :show-back="false">
     <EGButton
-      v-if="useUserStore().canCreateLab()"
+      v-if="useUserStore().canCreateLab(useUserStore().currentOrgId)"
```

Comment:
Given the org id isn't ever changing, rather than pass `useUserStore().currentOrgId` as an arg in all of these calls, could it instead be referenced directly in the method? e.g. 

```v-if="useUserStore().canCreateLab()"```

```
    canCreateLab:
      (_state: UserStoreState): boolean =>
        !useUserStore().isSuperuser && useUserStore().isOrgAdmin(useUserStore().currentOrgId)
```

Similarly for canManageOrgs, canEditLabUsers, etc.

Reply:
That's what we're coming from. I changed it to anticipate implementing proper org multitenancy, as we will need to specify the org when that happens, so the current status of the org id never changing is a temporary situation. Also, all the other functions below in this file take orgId as a parameter (as relevant) so this change brought these two functions in line with them.

I don't feel too strongly about this, and if we want to use currentOrgId within these permissions functions and not take orgId as a parameter, I'd be inclined to make the complete opposite change and get rid of the orgId parameter in all the other functions where it currently appears, so things are consistent.

Thoughts?

Reply:
Ah yep, I remember now. 

How about making the arg optional but default it's value to the current org if not passed in?

Reply:
I'm not aware of a good way to do that with Typescript, because optional orgId would come before mandatory labId, which creates a clunky function signature.

I think omitting orgId everywhere is fine too. It makes the code easier for now, and re-adding it will just be a migration task for when we implement multitenancy (there will be migration work for that anyway).

Sound good to you @simondib ?

Reply:
> I'm not aware of a good way to do that with Typescript, because optional orgId would come before mandatory labId, which creates a clunky function signature.

Are you referring to `canEditLabUsers`? If so you could just make the optional org id the last arg with a default value.

Reply:
Are we ever going to want to specify orgId before multitenancy is implemented anyway? Not sure it adds anything that we will use.

Reply:
I think the optional approach gives flexibility, but I'll leave it with you to decide.. it's not a hill I'm willing to die on :sweat_smile:

Reply:
Yeah nah, me either, especially as it's a temporary implementation 👍 

We can change it any time if needed.